docs/20_smart-contracts/10_getting-started/20_accounts/index.md

```diff
@@ -0,0 +1,208 @@
+
+The mandatory permissions are `owner` and `active`.
+
+A permission can only ever change its own information or the information of its children. It can never change its parent’s information.
+
+![Who can change permissions?](images/who_can_change_permissions.png)
+
+
+The information that controls a permission is either another `account@permission` or a public key. This allows for the creation of complex account control structures, where multiple parties can have control over a single account while having full autonomy over their own account’s security. 
+
+Take the following diagram as an example, where the account `alice` is controlled both by `bob` and `charlie`, while `charlie` is also controlled by `tom`. Eventually, all accounts are controlled by public keys. 
+
+
+![Delegated account ownership](images/delegated_account_ownership.png)
+
+
+You can add custom permissions underneath `active` which allows you to limit that permission’s access to only a specific contract’s action (callable function). That permission will then only ever be able to interact with the contract action you specified. 
+
+This means you are able to create granular access permissions across accounts and have hierarchical ownership and usage of them. 
+
+
+![Custom permissions](images/custom_permissions.png)
+
+
+Most importantly, the permission system has built-in support for multi-signature transactions (transactions that require multiple parties to sign them). Every linked account or key associated with a permission has a **weight** assigned to it, and the permission itself has a **threshold**. 
+
+As you can see in the example below, `bob` alone does not have enough power to sign using the `active` permission. He needs either `charlie` or `jenny` to co-sign with him for any transaction that `alice@active` makes. 
+
+
+![Weights and thresholds](images/weights_and_thresholds.png)
+
+
+## Smart Contracts
+
+A smart contract is just a program that runs on the blockchain. It allows you to add functionality to an account ranging from simple things like a todo application to a fully-fledged RPG game running entirely on the blockchain. 
+
+Every account has the ability to have one single smart contract deployed to it, however, that smart contract can be updated and replaced at will. 
+
+For more information about deploying smart contracts to your accounts, please see our **DUNE Guide**. 
+
+
+## Creating an Account with DUNE
+
+Once you have **DUNE** set up you can start creating accounts on your local development environment with a single command. 
+
+```
+dune --create-account <ACCOUNT_NAME>
+```
+
+If you want to see the information related to the account you just created, you can use the following command. 
+
+```
+dune -- cleos get account
+```
+
+
+## Ownership of Digital Assets
+
+When something is *owned* by an account on a blockchain, it means that a row in the decentralized database says that the digital asset belongs to that account. 

```

Comment:
Things stored on the blockchain are referred to as `digital assets`. When a digital asset is *owned* by an account......